agent/tool-scripts/postprocess/mpstat-postprocess

```diff
@@ -59,11 +70,67 @@ sub calc_timestamp {
 	return $date_ms + $this_time;
 }
 
-# read the mpstat.txt and collect stats for all non dm-* devices
-open(MPSTAT_TXT, "$dir/mpstat-stdout.txt") || die "could not find $dir/mpstat-stdout.txt\n";
+sub cpu_is_online {
+	my $cpu = shift;
+	if (exists $online_cpus{$cpu}) {
+		return $online_cpus{$cpu};
+	} elsif (-e $dir . "/online-cpus.txt" and open(FH, $dir . "/online-cpus.txt")) {
```

Comment:
For the unit tests, should be add a `online-cpus.txt` file so that they won't rely on the random set of CPUs they'll detect?

Reply:
Fixed in PR #1212.